.travis.yml

```diff
@@ -13,11 +13,11 @@ services:
   - docker
 
 env:
-#  matrix:
-#    - TEST_SUITE=integration NODE_VERSION=8
-#    - TEST_SUITE=integration NODE_VERSION=10
-#    - TEST_SUITE=integration NODE_VERSION=12
-#    - TEST_SUITE=performance NODE_VERSION=12
+  matrix:
+    - TEST_SUITE=integration NODE_VERSION=8
+    - TEST_SUITE=integration NODE_VERSION=10
+    - TEST_SUITE=integration NODE_VERSION=12
+    - TEST_SUITE=performance NODE_VERSION=12
```

Comment:
The performance suite is failing because no tests are enabled. I think we should just skip it. 

https://travis-ci.org/github/medic/cht-core/jobs/747458060#L4937
```
Running "protractor:performance-tests-and-services" (protractor) task
API / Sentinel service port listening on 31337
Report destination:   tests/results/report.html
[23:12:35] E/configParser - Error code: 105
[23:12:35] E/configParser - Error message: Unknown test suite: performance
[23:12:35] E/configParser - Error: Unknown test suite: performance
```

Reply:
ahh yeah in my rushing to get the yaml fixed I missed re-commenting that bit out.